modules/v2x/common/v2x_proxy_gflags.cc

```diff
@@ -24,18 +24,55 @@
 namespace apollo {
 namespace v2x {
 
+//// IP & PORT
 DEFINE_string(grpc_client_host, "127.0.0.1", "grpc client host ip");
 DEFINE_string(grpc_server_host, "127.0.0.1", "grpc server host ip");
 DEFINE_string(grpc_client_port, "50100", "grpc client port num");
 DEFINE_string(grpc_server_port, "50101", "grpc server port num");
+DEFINE_string(grpc_debug_server_port, "50102", "grpc server debug port num");
+
+//// Channels
+DEFINE_string(localization_channel, "/apollo/localization/pose",
+              "localization channel name");
+DEFINE_string(perception_obstacle_channel, "/perception/obstacles",
+              "perception channel name");
+DEFINE_string(chassis_detail_channel, "/adu/carall", "chassis channel name");
+DEFINE_string(planning_channel, "/apollo/planning", "planning channel name");
```

Comment:
channel name gflags are  defined in https://github.com/ApolloAuto/apollo/blob/master/modules/common/adapters/adapter_gflags.h. Please consolidate them.